app/views/admin/bulk_assignments/new.html.erb

```diff
@@ -25,16 +23,13 @@
         <%= f.input :force_identifier_creation, as: :boolean, label: 'YES', wrapper_html: { class: "has-error" } %>
       </div>
     <% end %>
-
-    <div class="InfoCard-row">
+    <div class="IdentifierTypeSelection sc-m-v--s">
       <%= f.input :identifier_type_id, label: 'Identified By', collection: identifier_types, label_method: :name, value_method: :id, include_blank: true %>
     </div>
-    <div class="InfoCard-row VariantOptions">
+    <div class="VariantOptions">
```

Comment:
ok, we have in the past used `fs-` as a prefix for feature spec for these, but maybe something more explicit would be helpful, just a suggestion I think it's cool to leave alone too